assets/jsons/hotel_scenarios_config.json

```diff
@@ -0,0 +1,32 @@
+{
+  "seaside_resort": {
```

Comment:
mozna by te trzy hotelVisitProbabilities wsadzić do jednej tablicy hotel_visit_purpose i w json data loaderze zapisywać to do mapy gdzie klucz to typ hotelu a wartości to mapa o kluczu hotelvisitpurpose -> liczba